lib/product-import.coffee

```diff
@@ -342,7 +343,8 @@ class ProductImport
       @_getProductTypeIdByName(prodToProcess.productType.id)
     ])
     .then ([sameForAllAttributes, preparedProduct, newProductTypeId]) =>
-      synced = @sync.buildActions(preparedProduct, existingProduct, sameForAllAttributes)
+      console.log('@matchVariantsByAttr', @matchVariantsByAttr)
```

Comment:
Remove this

Reply:
https://github.com/sphereio/sphere-product-import/pull/180/commits/d5536a506d13ca1305ef9af04d22fd6b52e936a9